.github/workflows/auto-merge-bot.yml

```diff
@@ -1,8 +1,25 @@
 on: [pull_request_target, pull_request_review]
 name: Auto-Merge Bot
 jobs:
+  
+  await_ci:
+    runs-on: ubuntu-latest
+    name: Await Travis CI Success
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js Environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: await-ci
+        uses: alita-moore/EIP-Bot@await-ci-pass
+        id: await-ci
+        with:
+          GITHUB-TOKEN: ${{ secrets.TOKEN }} 
```

Comment:
What happens if CI fails?  Does this fail and leave us right back in the same situation, or does this pass and result in the next job running?

If the latter, consider renaming the branch to `await-ci`.  :)

Reply:
if  ci fails then the await-ci job fails but the next job (the bot) will not run

Reply:
also in that case it shouldn't be an issue because the bot shouldn't be auto-merging anyway if CI fails and a new commit ill be required, which will start from scratch